ur_controllers/include/ur_controllers/force_torque_sensor_controller.h

```diff
@@ -0,0 +1,49 @@
+//
+// Created by livanov93 on 12/12/20.
```

Comment:
If the license is clear I propose to add a proper license header.

Then we could just move this file into `ros2_controllers` repo